test/mysql-cdc-resumption/mzcompose.py

```diff
@@ -52,28 +52,30 @@
 def workflow_default(c: Composition) -> None:
     workflows_with_internal_sharding = [
         "disruptions",
+        # TODO(def-): Reenable when materialize#26127 is fixed
+        # "bin-log-manipulations",
+        # "short-bin-log-retention",
+    ]
+    # TODO(def-): Reenable when materialize#26127 is fixed
+    ignored_workflows = [
         "bin-log-manipulations",
         "short-bin-log-retention",
     ]
     sharded_workflows = workflows_with_internal_sharding + buildkite.shard_list(
-        [w for w in c.workflows if w not in workflows_with_internal_sharding],
+        [
+            w
+            for w in c.workflows
+            if w not in workflows_with_internal_sharding + ignored_workflows
```

Comment:
Why do we include `ignored_workflows` here?

Reply:
I removed this anyway, so nevermind